pkg/oc/bootstrap/clusterup/components/registry/registry_install.go

```diff
@@ -29,8 +30,6 @@ const (
 )
 
 type RegistryComponentOptions struct {
-	ClusterAdminKubeConfig *rest.Config
-
 	OCImage         string
```

Comment:
Can we call this OriginImage ?

Reply:
> Can we call this OriginImage

We expose it specifically because it is not the origin image.  This is overridden in OSE cases.